pages/customize-the-search-index.mdx

```diff
@@ -0,0 +1,109 @@
+
+<Section size="1">
+  <Text size="4" mt="9">
+    Under the hood Canopy uses
+    [FlexSearch](https://github.com/nextapps-de/flexsearch) to power the search
+    index. FlexSearch is a full-text, search library that is fast and easy to
+    use. It is also highly customizable. Canopy provides a number of ways to
+    customize the search index.
+  </Text>
+</Section>
+
+<Callout>
+  This guide assumes you have a Canopy IIIF project. See the [Create a
+  Project](/get-started) guide to get started.
+</Callout>
+
+## Use Case
+
+You'd like to use Canopy IIIF to create a digital exhibit featuring Arabic manuscripts. For example, this NUL Collection of [Arabic Manuscripts from West Africa](https://dc.library.northwestern.edu/collections/59ec43f9-a96c-4314-9b44-9923790b371c). The [IIIF Manifest data](https://api.dc.library.northwestern.edu/api/v2/search?query=collection.id:%2259ec43f9-a96c-4314-9b44-9923790b371c%22&as=iiif) contains both Arabic and English text in its `label` and `summary` properties.
+
+You'd like to customize search in the following three ways:
+
+- Support the "Arabic" character set in Search (in addition to default "Latin").
+- Include text from a Manifest's `summary` values in search results.
+- Include additional `Manifest` metadata in search results. In our example [Manifests](https://api.dc.library.northwestern.edu/api/v2/works/2ca1b09b-cbad-43dd-82bf-a7fa807269d8?as=iiif) we include "Contributor" and "Alternate Title" as `metadata` items and would like to surface these in search results.
+
+## Implementation
+
+<Steps>
+
+### Support additional language encoding(s)
+
+1. Open `/config/options.json` and add support for Arabic text.
+2. Add `"arabic:extra"` to the `search.flexSearch.charset` property.
+
+```json filename="config/options.json"
+"search": {
+    "flexSearch": {
+        ...
+      "charset": ["latin:extra", "arabic:extra"],
+        ...
+    }
+}
+```
+
+3. Restart your development server to see the changes.
+
+### Include summary in search results
+
+1. In `/config/options.json` update the value of `search.index.summary.enabled` to be `true`.
+
+```json filename="config/options.json"
+
+"search": {
+    "index": {
+        ...
+      "summary": {
+        "enabled": true
+      }
+    }
+}
+```
+
+2. Restart your development server to see the changes.
+
+### Customize Manifest metadata in search results
+
+1. Open `/config/canopy.json`
+2. Update both `dev.metadata` and `prod.metadata` to include the metadata fields you'd like exposed in search results. (In the [example Manifest](https://api.dc.library.northwestern.edu/api/v2/works/2ca1b09b-cbad-43dd-82bf-a7fa807269d8?as=iiif) we include "Contributor" and "Alternate Title" as `metadata` items.)
+
+```json filename="config/canopy.json"
+"dev": {
+    ...
+    "metadata": ["Date", "Subject", "Contributor", "Alternate Title"],
+    ...
+  },
+"prod": {
+    ...
+    "metadata": ["Date", "Subject", "Contributor", "Alternate Title"],
+    ...
+  },
+```
```

Comment:
I like adding this but man do I wish this was easier to explain. Let's chat about it in a bit?